Sources/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-add_compile_options("$<$<COMPILE_LANGUAGE:Swift>:SHELL:-package-name SourceKitLSP>")
+add_compile_options("$<$<COMPILE_LANGUAGE:Swift>:SHELL:-package-name sourcekit_lsp>")
```

Comment:
Why the switch?

Reply:
When building the tests using SwiftPM, this is the package name that SwiftPM will chose and it means that we can use declarations at the package level from the modules built using CMake.